docs/il-curriculum/il-curriculum-vitae-per-un-developer.md

```diff
@@ -0,0 +1,40 @@
+Se stai leggendo questo capitolo probabilmente ti stai guardando attorno, alla ricerca di un nuovo lavoro, una nuova collaborazione, una nuova RAL (si spera più alta dell'attuale).
+E, probabilmente, ti stai facendo anche un'altra domanda: davvero il CV si usa ancora? Ebbene sì! Il CV rimane lo strumento fondamentale (ma va ovviamente affiancato ad altri strumenti che vedremo in seguito) per proporsi a chi sta offrendo lavoro.
+Partiamo da un assunto fondamentale: il CV perfetto NON ESISTE, così come non esiste “il CV a prova di ATS” (Applicant Tracking System). 
+Le indicazioni che troverai in questo capitolo saranno squisitamente di tipo tecnico/qualitativo/estetico, per darti la possibilità di costruire un CV funzionale ed attrattivo, contenente il giusto mix di parole chiave, descrizione delle esperienze professionali, soft skill e molto altro. 
+
+Alcuni consigli utili prima di entrare nel vivo:
+
+- È bene cercare di costruire il CV di una sola pagina (un/una recruiter esperto ci mette circa 10–20 secondi per leggerne uno) per catturare subito l’interesse e l’attenzione di chi riceverà il documento. _Content is king_, ma anche l’occhio vuole la sua parte!
+- Come vedrai nell’esempio di CV in basso il CV dovrebbe essere sintetico ma esaustivo, contenente tutte le keyword necessarie per la posizione per la quale stai applicando
+- Occhio alla lingua nella quale è scritto l’annuncio! Se l’annuncio è scritto in inglese, è sempre meglio inviare il CV in inglese, se in italiano invialo in italiano.  Avere quindi due versioni del CV è sempre cosa buona e giusta
+- Personalizza! L'europass è un formato ormai datato e senza alcuna possibilità di personalizzazione grafica. L'esempio che troverai in basso è stato creato con [Canva](https://www.canva.com/), contenente centinaia di template validi che puoi scegliere.
+  Potresti usare anche [LaTeX](https://www.latextemplates.com/) oppure [OverLeaf](https://www.overleaf.com/))
+
+Aggiungo inoltre alcune considerazioni più "personali" sulle quali dovresti riflettere, prendendo spunto da un post di Roberta Zantedeschi, che trovi [QUI]
+(https://www.linkedin.com/posts/robertazantedeschi_cv-curriculum-parolepersonelavoro-activity-7034528956879974400-kGoE?utm_source=share&utm_medium=member_desktop)
+
+Quali sono i principali ostacoli che potresti incontrare quando scrivi il tuo CV?
+- in primis poca connessione e conoscenza di te
+- scarsa stima del tuo valore e delle tue risorse
+- idee poco chiare rispetto a ciò che stai cercando, a chi stai parlando, alla direzione che vuoi perseguire
+- urgenza (scriverlo quando sei nel bisogno e non nella presenza a te e ai tuoi obiettivi)
+- poca cura della struttura, della gerarchia delle informazioni e della componente grafica e visual
+- difficoltà a selezionare le informazioni che servono (scartando quelle che non aggiungono valore)
+- convinzione talebana che il CV debba stare in una pagina a tutti i costi
```

Comment:
@GuidoPenta Questa frase siamo sicuri di volerla scrivere? (tenendo conto del CODE OF CONDUCT che abbiamo redatto [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md))